working-groups/green-reviews/README.md

```diff
@@ -0,0 +1,33 @@
+# Green Reviews Working Group
+
+The Green Reviews [Working Group](https://github.com/cncf/toc/tree/main/workinggroups) operates under CNCF Environmental Sustainability TAG.
+
+The Green Reviews Working Group helps CNCF projects assess and improve their cloud native sustainability footprint. By integrating environmental sustainability reviews into release cycles and empowering end users to evaluate project sustainability, the group promotes long-term viability and environmental responsibility within the CNCF ecosystem.
+
+See the working group charter here: <https://github.com/cncf/tag-env-sustainability/tree/main/working-groups/green-reviews/charter.md>
+
+## Motivation
+
+CNCF projects are becoming increasingly concerned and aware of the challenges related to environmental sustainability. Their objective is to integrate an environmental sustainability project review in each release cycle. Furthermore, they strive to empower end users to assess the sustainability of the cloud native projects within their systems. 
+
+The purpose of the Green Reviews Working Group is to help CNCF projects to review their cloud native sustainability footprint.
+
+To support CNCF projects, project maintainers and contributors, there is a need for a systematic approach to assess every project's sustainability footprint. Implementing a structured review process for this purpose would greatly benefit the CNCF ecosystem, establishing it as a best practice. Such a process would enable stakeholders to comprehensively evaluate and enhance the sustainability of CNCF projects, promoting their long-term viability and environmental responsibility. 
+
+Green Reviews Working Group empowers the CNCF Environmental Sustainability TAG to pursue its goals of promoting environmentally sustainable practices in the CNCF ecosystem.
+
+## Chairs
```

Comment:
Since this WG will persist, @catblade @mkorbi we need to how long leads serve in this role and how followup elections look like.

Reply:
It makes sense for there to be a commitment as a lead.  I'd recommend we do at least 6 months.  @TheFoxAtWork  what is the normal cadence of assignments for leads of WGs?

Reply:
Yes, thanks for bringing this up. Depending on what the process is we can either add an abstract describing this or refer to the same process that we have in the TAG if it can be re-used here as well. I can update the content once we get some additional information here.

Reply:
@leonardpahlke @catblade @nikimanoledaki @mkorbi I have added a document for the Working Group Chair nomination process as part of this PR where I used tech lead proposal doc as a foundation with further adjustments and simplifications for this specific use case - can you please take a look when you have time and let me know if it requires any further adjustments? Thanks! 😊

Reply:
LGTM